src/lib/fragments/accordion.css

```diff
@@ -0,0 +1,8 @@
+.accordion-button:focus,
```

Comment:
this looks like it's from the other PR. Maybe merge main into this branch?

Reply:
Good idea